webapp/channels/src/utils/constants.tsx

```diff
@@ -29,16 +29,10 @@ import {t} from 'utils/i18n';
 
 import {CustomStatusDuration} from '@mattermost/types/users';
 
-import githubCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/github.css';
-
-// eslint-disable-line import/order
-import monokaiCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/monokai.css';
```

Comment:
how did we solve the webpack not stripping away these css files issue?

Reply:
oh we added a new rule

Reply:
Yeah, I put it in the webpack config. I didn't actually understand that this was calling out to the Webpack loader directly before, but that's harder to do with asset modules